src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Func;
+use Cron\CronExpression;
+use Utopia\Database\Database;
+use Utopia\Pools\Group;
+
+class ScheduleExecutions extends ScheduleBase
+{
+    public const UPDATE_TIMER = 3; // seconds
+    public const ENQUEUE_TIMER = 4; // seconds
```

Comment:
Those numbers look very random. What about 5 and 5? 3 seems a bit too fast, considering we might have many of those worker, causing bunch of SQL queries just from here.

Reply:
These numbers are copied from messaging scheduling, which works well. I don't think `5` and `3` will offer any difference?